### PR TITLE
Update to the 22000 Microsoft.Windows.SDK.BuildTools nuget package

### PR DESCRIPTION
**Pull request by evelynwu-msft**
The 22000 BuildTools package contains a fix that will address this issue: https://github.com/microsoft/microsoft-ui-xaml/issues/5828. This PR is paired with PR 6532299 in the internal repo.

### CODE REVIEW

File: dev/VSIX/Directory.Build.props [hunk 1]
```diff
@@ -7,7 +7,7 @@
             https://pkgs.dev.azure.com/microsoft/ProjectReunion/_packaging/Project.Reunion.nuget.internal/nuget/v3/index.json
         </RestoreSources>
         <CppWinRTVersion Condition="'$(CppWinRTVersion)' == ''">2.0.210806.1</CppWinRTVersion>
-        <WindowsSDKBuildToolsVersion Condition="'$(WindowsSDKBuildToolsVersion)' == ''">10.0.20348.19</WindowsSDKBuildToolsVersion>
+        <WindowsSDKBuildToolsVersion Condition="'$(WindowsSDKBuildToolsVersion)' == ''">10.0.22000.194/WindowsSDKBuildToolsVersion>
```
Comment (asklar) [inline dev/VSIX/Directory.Build.props:10]: typo:
```suggestion
        <WindowsSDKBuildToolsVersion Condition="'$(WindowsSDKBuildToolsVersion)' == ''">10.0.22000.194>/WindowsSDKBuildToolsVersion>
```

**Review by asklar — APPROVED** [review]
thanks @evelynwu-msft ! there's a typo but LGTM otherwise

File: dev/VSIX/Directory.Build.props (hunk 1 above)
Comment (rohanp-msft) [reply dev/VSIX/Directory.Build.props:10]: Fix: https://github.com/microsoft/WindowsAppSDK/pull/1537

Comment (rohanp-msft) [reply dev/VSIX/Directory.Build.props:10]: Merged

Comment (evelynwu-msft) [reply dev/VSIX/Directory.Build.props:10]: Good catch @asklar, and thanks @rohanp-msft!

**Pull request merged**